### PR TITLE
Fix errors in pruning commands

### PR DESCRIPTION
**Pull request by jboxman**
- https://github.com/openshift/openshift-docs/issues/22258

### CODE REVIEW

File: modules/pruning-images.adoc [hunk 1]
```diff
@@ -20,12 +20,12 @@ When the Pruning Custom Resource is deleted, the pruning `CronJob` and its relat
...
 .Procedure
 
-. Verify that the object named `imagepruners.imageregistry.operator.openshift.io/cluster` contains the following `spec` and `status` fields:
+* Verify that the object named `imagepruners.imageregistry.operator.openshift.io/cluster` contains the following `spec` and `status` fields:
 
 [source,yaml]
 ----
 spec:
-  schedule: "*/0 * * * *" <1>
+  schedule: "0 0 * * *" <1>
```
Comment (wzheng1) [inline modules/pruning-images.adoc:28]: Acked by QE.

File: modules/pruning-groups.adoc [hunk 2]
```diff
@@ -35,13 +35,13 @@ $ oc adm prune groups \
 To see the groups that the prune command deletes:
 
 ----
-$ oc adm prune groups --sync-file=ldap-sync-config.yaml
+$ oc adm prune groups --sync-config==ldap-sync-config.yaml
```
Comment (rhpmali) [inline modules/pruning-groups.adoc:38]: Please remove one `=` from `--sync-config==` command

File: modules/pruning-groups.adoc [hunk 3]
```diff
@@ -35,13 +35,13 @@ $ oc adm prune groups \
...
 ----
 
 To perform the prune operation:
 
 ----
-$ oc adm prune groups --sync-file=ldap-sync-config.yaml --confirm
+$ oc adm prune groups --sync-config==ldap-sync-config.yaml --confirm
```
Comment (rhpmali) [inline modules/pruning-groups.adoc:44]: Please remove one `=` from `--sync-config==` command

**Review by rhpmali — CHANGES_REQUESTED** [review]

File: modules/pruning-images.adoc (hunk 1 above)
Comment (jboxman) [reply modules/pruning-images.adoc:28]: @wzheng1 thank you!

File: modules/pruning-groups.adoc (hunk 2 above)
Comment (jboxman) [reply modules/pruning-groups.adoc:38]: @rhpmali, oops, thanks!

**Review by rhpmali — COMMENTED** [review]
Edit: added comment in correct place

File: modules/pruning-groups.adoc [hunk 4]
```diff
@@ -35,13 +35,13 @@ $ oc adm prune groups \
...
 ----
-$ oc adm prune groups --sync-file=ldap-sync-config.yaml
+$ oc adm prune groups --sync-config=ldap-sync-config.yaml
 ----
 
 To perform the prune operation:
 
 ----
-$ oc adm prune groups --sync-file=ldap-sync-config.yaml --confirm
+$ oc adm prune groups --sync-config=ldap-sync-config.yaml --confirm
```
Comment (rhpmali) [inline modules/pruning-groups.adoc:44]: Acked by QE.

File: modules/pruning-groups.adoc [hunk 5]
```diff
@@ -35,13 +35,13 @@ $ oc adm prune groups \
 To see the groups that the prune command deletes:
 
 ----
-$ oc adm prune groups --sync-file=ldap-sync-config.yaml
+$ oc adm prune groups --sync-config=ldap-sync-config.yaml
```
Comment (rhpmali) [inline modules/pruning-groups.adoc:38]: Acked by QE.

**Pull request merged**